config/_files.yml

```diff
@@ -54,6 +54,7 @@ bootstrap:
     javascriptEsmSri: sha384-sKZy8g2KJhBTFCD6cIg8d4EifJxaa8c/iYIERdeKorHWhAgZgQOfqOKMe3xBqye1
```

Comment:
I think this file should be removed completely from the repo. For auto-updates to work we need to generate it during deployment. I dont see a reason to host it in the repo

Reply:
I can put it in gitignore

Reply:
Do everything you need to delete it from the repo and make sure the tests run on the file that is generated on the fly